cl/search/es_indices.py

```diff
@@ -24,3 +24,12 @@
     number_of_replicas=settings.ELASTICSEARCH_OA_ALERTS_NUMBER_OF_REPLICAS,
     analysis=settings.ELASTICSEARCH_DSL["analysis"],
 )
+
+
+# Define people elasticsearch index
+people_db_index = Index("people_db_index")
+people_db_index.settings(
+    number_of_shards=settings.ELASTICSEARCH_NUMBER_OF_SHARDS,
+    number_of_replicas=settings.ELASTICSEARCH_NUMBER_OF_REPLICAS,
```

Comment:
We want distinct settings here, don't we? Like we did for OA?

Reply:
Sure, I've added custom settings for People.